readthedocs/search/tasks.py

```diff
@@ -209,3 +210,29 @@ def record_search_query(project_slug, version_slug, query, total_results, time_s
         version=version,
         query=query,
     )
+
+
+@app.task(queue='web')
+def increase_page_view_count(project_slug, version_slug, path):
```

Comment:
I wonder if running this as a task is actually a good deal slower then running it in the footer itself? We already have the Project & Version object in the footer, and not we're adding additional queries for them here. They should be fast queries, but definitely additional load. 

I guess the proper solution is to cache these lookups, which we can do later if it's an issue.